ops/README.md

```
@@ -38,11 +45,32 @@ data/MC_st0r5.2_6.dat
 
 ### Panel manifest
 
-`ops/gates/panel_manifest.tsv` is tab-separated: `pdb <TAB> receptor.pdb <TAB> native_pose.sdf <TAB> decoy_pose.pdb`
+`ops/gates/panel_manifest.tsv` is tab-separated (5 fields):
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)